.ci/snapshot.sh

```diff
@@ -24,9 +24,14 @@ echo $PATH
 java -version
 
 set +x
-echo "--- Deploy the snapshot :package:"
-if [[ "$dry_run" == "true" ]] ; then
-  ./mvnw -V -s .ci/settings.xml -Pgpg clean package --batch-mode | tee snapshot.txt
-else
-  ./mvnw -V -s .ci/settings.xml -Pgpg clean deploy --batch-mode | tee snapshot.txt
+# Default in dry-run mode
+GOAL="install"
```

Comment:
For dry-run, `package` is enough

```suggestion
GOAL="package"
```